internal/controllers/clusterclass/clusterclass_controller_test.go

```diff
@@ -1146,8 +1146,8 @@ func TestReconciler_reconcileVariables(t *testing.T) {
 							OpenAPIV3Schema: clusterv1.JSONSchemaProps{
 								Type: "string",
 								XValidations: []clusterv1.ValidationRule{{
```

Comment:
Note to myself: this should be fixed in a way that we don't have to bump these tests on every k/k minor release (cf with how k/k is solving it)